README.md

```diff
@@ -20,8 +20,9 @@ Features:
   * [node-xml](https://github.com/dylang/node-xml)
   * [node-expat](https://github.com/node-xmpp/node-expat)
   * [libxmljs](https://github.com/polotek/libxmljs)
-  * [ltx](https://github.com/node-xmpp/ltx/blob/master/lib/parsers/ltx.js) (default)
-* [JSX](https://facebook.github.io/jsx/) compatible
+  * [ltx](https://github.com/node-xmpp/ltx/blob/master/lib/parsers/ltx.js) (default and fastest see [Benchmarks](#benchmarks))
+* [JSX](https://facebook.github.io/jsx/) compatible (use `ltx.createElement` pragma)
+* [tagged template](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings) support `` ltx`<foo bar="${baz}">` ``
```

Comment:
BTW we should probably automatically add missing `"` when detecting an attribute in tagged templates